packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -0,0 +1,526 @@
+---
+title: Multi-tenant solutions
+---
+
+<div class="multi-tenancy">
+
+# Multi-tenant solutions
+
+<div class="title-subtext">
+What is multi-tenancy? Does my company need it? If so, how should it
+be configured to meet my company’s needs?
+</div>
+
+## Acme Bank example
+
+<section class="acme-bank-container">
+<div class="acme-bank-text">
+Acme Bank is a fictitious banking company which had come to Okta to
+centralize and improve their existing identity management infrastructure.
+To meet the needs and demands of their employees, customers, and partners,
+they built their solution with a multi-tenant configuration. Acme Bank is
+used through this guide to explain details and key concepts around
+multi-tenancy.
+</div>
+<div class="acme-bank-image">
+
+![Acme Bank Logo](/img/multi-tenancy/acme-bank-logo.png "The logo of Acme Bank")
+
+</div>
+</section>
+
+## What is a tenant?
+A tenant is a single instance of software and supporting infrastructure that
+supports a group of users. These groups can represent any entity that
+interfaces with the organization such as its employees, customers or partners.
+This entity can be an individual user (B2C) or another company (B2B) with its
+own set of users.
+
+### A tenant within Identity Access Management
+A tenant can also be considered as an isolated island of data that is separate
+and private from other tenants. In the context of identity management, each
+tenant has its own security policies, user registration settings, user groups,
+roles and rules that determine application access. An illustration of these
+different types of data are shown below:
+
+<div class="image-at-70">
+
+![What is a tenant?](/img/multi-tenancy/what-is-a-tenant.png "Examples of Acme Bank's tenants")
+
+</div>
+
+## Why would you want more than one tenant
+
+An organization can create a new tenant for a variety of reasons. For example
+they may want to:
+
+<div class="modern-list">
+
+* Separate out user administration and application access for internal employees
+  from outside customers.
+* Store customer data in the country of origin due to regulations and data
+residency requirements.
+* Satisfy a unique set of requirements (e.g. distinct branding emails,
+onboarding videos) for a business with their own set of users.
+* Increase isolation, performance, and scalability for an organization with a large
+user base.
+
+</div>
+
+## Types of tenants
+
+Tenants can comprise of one or more of the following entities:
+
+<div class="modern-numbered-list">
```

Comment:
I don't see why this list needs to be numbered at all.

Also, because this is being done with CSS, if you're just reviewing the markdown it's easy to miss that this is a numbered list in the first place.

Reply:
The styling of the lists has been removed.  Regarding the "miss" comment, the "modern-numbered-list" identifies that the list is a number.  This is how I've done numbered lists in the past.  I will research however, alternatives ways such as formatting a numbered list to make it more explicit.